avatar-screenshotter.js

```diff
@@ -84,10 +84,30 @@ export const screenshotPlayer = async ({
       _animate(now, timeDiff);
     };
   };
+  const localVector = new THREE.Vector3();
   const _updateTarget = (timestamp, timeDiff) => {
+    const neckPosition = localVector.setFromMatrixPosition(player.avatar.modelBones.Head.savedMatrixWorld);
+    let avatarHighestPos = 0;
+    let tempMesh = null;
+    player.avatar.model.traverse(o => {
+      if (o.isMesh) {
+        for(let i = 0; i < o.geometry.attributes.position.array.length / 3; i++){

```

Comment:
This is extremely slow.

Instead of this, can we use the physics bounding box or some other heuristic? We can't check all vertices.